### PR TITLE
Restore UTF8 to UTF16 convertion

### PR DESCRIPTION
**Pull request by eugeneo**

### CODE REVIEW

File: platform/inspector_protocol/String16STL.cpp [hunk 1]
```diff
@@ -285,6 +485,23 @@ static inline void putUTF8Triple(char*& buffer, UChar ch)
...
+String16 String16::fromUTF8(const char* stringStart, size_t length)
+{
+    if (!stringStart || !length)
+        return String16();
+
+    std::vector<UChar> buffer(length);
+    UChar* bufferStart = buffer.data();
+
+    UChar* bufferCurrent = bufferStart;
+    const char* stringCurrent = reinterpret_cast<const char*>(stringStart);
```
Comment (pavelfeldman) [inline platform/inspector_protocol/String16STL.cpp]: no need to reinterpret.

File: platform/inspector_protocol/String16STL.cpp [hunk 2]
```diff
@@ -285,6 +485,23 @@ static inline void putUTF8Triple(char*& buffer, UChar ch)
...
+    if (convertUTF8ToUTF16(&stringCurrent, reinterpret_cast<const char *>(stringStart + length), &bufferCurrent, bufferCurrent + buffer.size(), 0, true) != conversionOK)
```
Comment (pavelfeldman) [inline platform/inspector_protocol/String16STL.cpp]: ditto

**Comment by eugeneo** [comment]
Done.

**Comment by pavelfeldman** [comment]
LGTM

**Pull request merged**